state-chain/pallets/cf-ingress-egress/src/migrations.rs

```diff
@@ -1,4 +1,8 @@
 use crate::Pallet;
-use cf_runtime_upgrade_utilities::PlaceholderMigration;
+use cf_runtime_upgrade_utilities::{PlaceholderMigration, VersionedMigration};
+mod remove_deposit_tracker;
 
-pub type PalletMigration<T, I> = PlaceholderMigration<Pallet<T, I>, 12>;
+pub type PalletMigration<T, I> = (
+	VersionedMigration<Pallet<T, I>, remove_deposit_tracker::Migration<T, I>, 12, 13>,
+	PlaceholderMigration<Pallet<T, I>, 13>,
+);
```

Comment:
Looks like something might have got lost/mixed up in a merge - pallet version is still 12.